dotcom-rendering/src/web/components/KeyEventCard.tsx

```diff
@@ -90,7 +90,12 @@ export const KeyEventCard = ({ text, date, url, format }: KeyEvent) => {
 
 	return (
 		<li css={listItemStyles}>
-			<Link priority="secondary" css={linkStyles} href={url}>
+			<Link
+				priority="secondary"
+				css={linkStyles}
+				href={url}
+				data-link-name="key event card"
```

Comment:
Oh, it's a prop on `Link`, nice!

Reply:
`Link` has a wildcard `...props` input which is then spread onto the `a` tag - works nicely for this :)